src/packages/solid/components/Input/Input.styles.ts

```diff
@@ -63,7 +64,7 @@ const input: InputConfig = {
     contentColor: 'contentColor'
   },
   base: {
-    width: 400,
+    width: getWidthByUpCount(theme, 4),
```

Comment:
hmmm. this does work but it's not really themeable, at least not via componentConfig

Reply:
this was how it was in LUI5, is there a different theme value I should use?

Reply:
its fine for now, i have feelings about it but it might just stay this way